### PR TITLE
Make tests pass

### PR DESCRIPTION
**Pull request by WolfByttner**

### CODE REVIEW

File: traja/tests/test_nn.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 data_url = "https://raw.githubusercontent.com/traja-team/traja-research/dataset_und_notebooks/dataset_analysis/jaguar5.csv"
 df = pd.read_csv(data_url, error_bad_lines=False)
-model_save_path = './model.pt'
+model_save_path = 'model'
```
Comment (Saran-nns) [inline traja/tests/test_nn.py]: I think, saving as str 'model' without extension will be an issue.  how about keeping the .pt extension?

Comment (Saran-nns) [inline traja/tests/test_nn.py]: Or you recommend to create dir "model" to save the model as "model.pt" by default?

**Review by Saran-nns — CHANGES_REQUESTED** [review]

File: traja/tests/test_nn.py (hunk 1 above)
Comment (WolfByttner) [reply traja/tests/test_nn.py]: That file is replaced by of test_model.py - issue fixed there!

Comment (WolfByttner) [reply traja/tests/test_nn.py]: That file is replaced by of test_model.py - issue fixed there!

**Review by Saran-nns — APPROVED** [review]

**Pull request merged**